autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -127,6 +128,25 @@ public void validateClusterLimit(AdjustmentType adjustmentType, Integer configur
         }
     }
 
+    public void validateScheduleWithStopStartAndDisable(Cluster cluster, DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
+        if (!autoscaleClusterRequest.getTimeAlertRequests().isEmpty() &&
+                (Boolean.TRUE.equals(autoscaleClusterRequest.getUseStopStartMechanism()) || Boolean.TRUE.equals(cluster.isStopStartScalingEnabled()))) {
```

Comment:
The entire || block supposed to be a single statement? The ")" closes after the first check.

Reply:
Updated this condition to be more readable and also fixed a flaw in the logic